docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+version: '3.3'
+
+services:
+
+  reverse-proxy:
+    # The official v2.1 Traefik docker image
+    image: traefik:v2.1
+    restart: always
+    # Enables the web UI and tells Traefik to listen to docker
+    command:
+#      - "--log.level=DEBUG"
+      - "--api.insecure=true"
+      - "--providers.docker=true"
+      # You need to actively say traefik.enable=true
+      - "--providers.docker.exposedbydefault=false"
+      - "--entrypoints.websecure.address=:443"
+      - "--certificatesresolvers.mytlschallenge.acme.tlschallenge=true"
+      - "--certificatesresolvers.mytlschallenge.acme.email=nick@nicholasyang.com"
+      - "--certificatesresolvers.mytlschallenge.acme.storage=/letsencrypt/acme.json"
+    ports:
+      - "80:80"
+      - "443:443"
+      # The Web UI (enabled by --api.insecure=true)
+      - "8080:8080"
+    volumes:
+      # So that Traefik can listen to the Docker events
+      - "/var/run/docker.sock:/var/run/docker.sock:ro"
+      - "./letsencrypt:/letsencrypt"
+    labels:
+      # Redirects all traffic to HTTPS
+      - "traefik.http.routers.redirs.rule=hostregexp(`{host:.+}`)"
+      - "traefik.http.routers.redirs.entrypoints=web"
+      - "traefik.http.routers.redirs.middlewares=redirect-to-https"
+
+  db:
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_PASSWORD:
+      POSTGRES_DB: schedge
+      POSTGRES_USER: schedge
+
+  app:
+    build: .
+    environment:
+      JDBC_URL: jdbc:postgresql://db:5432/schedge
+    ports:
+      - "8001:80"
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.app.rule=Host(`albertsucks.com`)"
```

Comment:
I feel like having it hosted at `albersucks.com` is a bit rude. Not really my style and definitely not BUGS's style. Can we stick to the current domain or switch to a schedge-specific one?

Reply:
Smh, NYU Albert does suck. Not you buddy :D. Switched to schedge.torchnyu.com